app/views/feedbacks/thanks.html.erb

```diff
@@ -0,0 +1,50 @@
+<html>
+  <head>
+    <title>Thank you for your feedback</title>
+    <style>
```

Comment:
Why you use styles for only one mailer view? How about use in general? And it is better to define classes in css files, not in `<style>` tag

Reply:
OK, I do this in another PR for view part